app/assets/v2/js/pages/dashboard.js

```diff
@@ -393,6 +393,36 @@ var reset_offset = function() {
   document.offset = 0;
 };
 
+// function that resurfaces an issue to the top when expiration has been extended
+var resurfaceExtendedExpirationIssue = function(issues) {
```

Comment:
🤔 Any reason we're not updating the backend expiration date and allowing the backend to resurface the issue in the appropriate order?

Reply:
@mul1sh  This will probably break when more results are loaded in from the paginated results coming from the backend, since order is dictated by `order_by` param being passed to the API and if it doesn't, it'll have to run through all results as the user scrolls down and attempts to render more results.  If we want to display any kind of visual indicator that the expiration was recently updated, this is something we could also determine in the router and pass as a key/value in the response.

We probably don't want to manipulate the `order_by` results of the queried data on the frontend.  Since the expiration is being updated, why not just adjust the filtering on the explorer from `-created` to `-modified_on` ?  Plus this might hinder performance, since we're passing all results coming from the backend into this function.

Reply:
https://github.com/gitcoinco/web/pull/2256#issuecomment-423342065